packages/client/src/users/types/guestUser.types.ts

```diff
@@ -0,0 +1,57 @@
+export type GuestUserResponse = {
+  id: 0;
+  bagId: string;
+  wishlistId: string;
+  ip: string;
+  countryCode: string;
+  externalId: string;
+  friendId: string;
+  expiryDate: string;
+};
+
+export type PreferencesResponse = [
+  {
+    code: string;
+    values: string[];
+    groupId: string;
+    updatedDate: string;
+  },
+];
+
+export type PostGuestUserData = {
+  countryCode: string;
+  ip: string;
+  externalId: string;
+  friendId: string;
+};
+
+export type SetPreferencesData = [
+  {
+    code: string;
+    values: string[];
+    groupId: string;
+    updatedDate: string;
+  },
+];
+
+export type PostGuestUser = (
```

Comment:
There's a few more to move to individual files :eyes:

Reply:
Fixed 👍🏻